kubernetes/ama-logs.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v1
```

Comment:
Please get the rename updates for the addon-VPA-resizer.   Please marge latest dev and double check if there are no updates which requires rename updates.

Reply:
done